From 1605a69f29e5de2831e7f54367fa925fcc080903 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:41:17 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(4ef0a6b)..lineage-16.0-android-9.0.0_r31(0a4a41b) (DO NOT
 SUBMIT)

Android Build Merger (Role) (2):
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   7997e44a77
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   7997e44a77 skipped: bfa15bcbdf

Bill Yi (5):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   7997e44a77 skipped: bfa15bcbdf am: 3d37465e84  -s ours
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   7997e44a77 skipped: bfa15bcbdf am: 3d37465e84  -s ours am: acfd17bba2
    -s ours
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   7997e44a77 skipped: bfa15bcbdf am: 3d37465e84  -s ours am: acfd17bba2
    -s ours am: 6bc9557538  -s ours

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (2):
 - Snap for 5073521 from 02bd7c156e3a0e27623b2d978501784f095fa1f9 to
   pi-qpr2-release
 - Snap for 5130385 from c78fc83e718dff47b64e33577cc6de042ce23bba to
   pi-qpr2-release

Change-Id: I5bd019840a16976fa70343986cf1d250502f62f9

Change-Id: I507691b7d46d497b05ff63d0b0decce73c676d75
---
 res/values-de/strings.xml | 2 +-
 res/values-hr/strings.xml | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/res/values-de/strings.xml b/res/values-de/strings.xml
index cc9af49..a75c831 100644
--- a/res/values-de/strings.xml
+++ b/res/values-de/strings.xml
@@ -74,5 +74,5 @@
     <string name="wifi_installer_fail" msgid="894483173306576331">"Die WLAN-Anmeldedaten können nicht installiert werden. Lade die Datei erneut herunter."</string>
     <string name="wifi_installer_fail_no_wifi_title" msgid="8358191074828840533">"Installation abgebrochen"</string>
     <string name="wifi_installer_fail_title" msgid="5547079779067835479">"Installation nicht möglich"</string>
-    <string name="wifi_installer_fail_no_wifi" msgid="2044576439984209921">"Aktiviere WLAN und versuche es erneut."</string>
+    <string name="wifi_installer_fail_no_wifi" msgid="2044576439984209921">"Aktiviere WLAN und versuche es noch einmal."</string>
 </resources>
diff --git a/res/values-hr/strings.xml b/res/values-hr/strings.xml
index f4482b5..d6d42e2 100644
--- a/res/values-hr/strings.xml
+++ b/res/values-hr/strings.xml
@@ -53,7 +53,7 @@
     <item msgid="692322974985472861">"Wi-Fi"</item>
   </string-array>
     <string name="certificate_capabilities_warning" msgid="4158609401937799110">"Napomena: izdavatelj tog certifikata može pregledavati sav promet upućen s uređaja i na uređaj."</string>
-    <string name="wifi_title" msgid="8475811746333426489">"Profil Wi-Fi-ja"</string>
+    <string name="wifi_title" msgid="8475811746333426489">"Profil Wi-Fija"</string>
     <string name="wifi_detail_title" msgid="3627332137252994395">"Pojedinosti za %s"</string>
     <string name="wifi_detail_label" msgid="3032151019356747583">"Pojedinosti"</string>
     <string name="wifi_install_label" msgid="1449629407724323233">"Instaliraj"</string>
@@ -69,9 +69,9 @@
     <string name="install_done_title" msgid="4822989918460334497">"Vjerodajnice su instalirane"</string>
     <string name="install_done" msgid="1522718684172097080">"<xliff:g id="NAME">%1$s</xliff:g> vjerodajnice dodane su u spremljene Wi-Fi mreže."</string>
     <string name="done_label" msgid="5597425930652904250">"GOTOVO"</string>
-    <string name="wifi_installer_detail" msgid="63879632832701669">"Instalirajte vjerodajnice Wi-Fi-ja da biste se povezivali s mrežama dostupnim putem <xliff:g id="NAME">%1$s</xliff:g>."</string>
+    <string name="wifi_installer_detail" msgid="63879632832701669">"Instalirajte vjerodajnice Wi-Fija da biste se povezivali s mrežama dostupnim putem <xliff:g id="NAME">%1$s</xliff:g>."</string>
     <string name="wifi_installer_download_error" msgid="5429453090956277692">"Na preuzetoj datoteci ima problema, pa je nije moguće instalirati. Provjerite jeste li preuzeli datoteku iz točnog izvora."</string>
-    <string name="wifi_installer_fail" msgid="894483173306576331">"Instaliranje vjerodajnica Wi-Fi-ja nije uspjelo. Ponovo pokušajte preuzeti datoteku."</string>
+    <string name="wifi_installer_fail" msgid="894483173306576331">"Instaliranje vjerodajnica Wi-Fija nije uspjelo. Ponovo pokušajte preuzeti datoteku."</string>
     <string name="wifi_installer_fail_no_wifi_title" msgid="8358191074828840533">"Instalacija otkazana"</string>
     <string name="wifi_installer_fail_title" msgid="5547079779067835479">"Instaliranje nije moguće"</string>
     <string name="wifi_installer_fail_no_wifi" msgid="2044576439984209921">"Uključite Wi-Fi i pokušajte ponovo."</string>
-- 
2.17.1

